packages/blockchain-extension/test/commands/deleteEnvironmentCommand.test.ts

```diff
@@ -123,8 +125,8 @@ describe('DeleteEnvironmentCommand', () => {
 
             await vscode.commands.executeCommand(ExtensionCommands.DELETE_ENVIRONMENT);
 
-            showFabricEnvironmentQuickPickBoxStub.should.have.been.calledOnceWithExactly('Choose the environment(s) that you want to delete', true, false, true);
-            getAllSpy.should.have.been.calledOnceWithExactly(true);
+            showFabricEnvironmentQuickPickBoxStub.should.have.been.calledOnceWithExactly('Choose the environment(s) that you want to delete', true, false, []);
```

Comment:
I don't think you need `[]` in any of these either - as you're saying "get me all environments" (and the default values are []).